CHANGELOG.md

```diff
@@ -93,6 +93,17 @@ Other major additions
   Data.Nat.Binary.Subtraction
   ```
 
+* Symmetry of various functional properties
+  ```agda
+  Function.Construct.Symmetry
+  ```
+
+* `IsEquivalence` bundles for `Inverse`, `Equivalence`, `↔` and `⇔` in
```

Comment:
`IsEquivalence` is a structure not a bundle?

Reply:
indeed.